Account.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+public class Account
+{
+	private string name;
```

Comment:
Looks like the indentation is a bit off in this file, looks like everything is indented more than it should? 🙂